src/statistics.jl

```diff
@@ -1,28 +1,46 @@
-# Autocorrelation for a specifig lag
-function autocor(x::AbstractVector, lag::Integer)
+"""
+    autocor(x,lag)
+
+Returns autocorrelation function for a distance of lag, i.e. between values of
+```x[t]``` and ```x[t+lag]```. The autocorrelation function is normalized such
+that ```autocor(x,0) = 1```. This is equvivalent to the quantity
+``\\Gamma_X(t)`` of equation (4.61) in Gattringer/Lang.
```

Comment:
Better citation of the book would be nice.
Is there anything in documenter.jl to have something like a "Reference"-section with Bibliography to link to?

Reply:
It turns out that there is a dedicated plug-in package for this [DocumenterCitations.jl](https://github.com/ali-ramadhan/DocumenterCitations.jl) . I had to modify the standard .bib entries generated by inspire for it to work and the bibliography style cannot be customized (yet) (see https://github.com/ali-ramadhan/DocumenterCitations.jl/issues/22 ) but this should do it for now.